package/opt/hassbian/suites/install_duckdns.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+function duckdns-show-short-info {
+echo -e "Setup for duckdns auto renewal"
```

Comment:
Duck DNS uses the following in their written material and we should try to follow it "Duck DNS".

Reply:
Done :)